test/integration/test_iam_identity_v1.py

```diff
@@ -1084,6 +1084,37 @@ def test_delete_link(self):
         link = self.get_link(self.iam_identity_service, profile_id2, link_id)
         assert link is None
 
+    @needscredentials
+    def test_delete_link_with_parameters(self):
+        CreateProfileLinkRequestLink = {}
+        CreateProfileLinkRequestLink['crn'] = (
+            'crn:v1:staging:public:iam-identity::a/' + self.account_id + '::computeresource:Fake-Compute-Resource'
+        )
+        CreateProfileLinkRequestLink['component_name'] = 'test_componenet_name'
+        CreateProfileLinkRequestLink['component_type'] = 'test_componenet_type'
```

Comment:
I _think_ this is s typo. :)

```suggestion
        CreateProfileLinkRequestLink['component_name'] = 'test_component_name'
        CreateProfileLinkRequestLink['component_type'] = 'test_component_type'
```